control_panel_api/tests/test_views.py

```diff
@@ -20,6 +20,26 @@
 )
 
 
+class AssertionsMixin(object):
+    def assert_s3bucket(self, s3bucket):
```

Comment:
These assertions are quite brittle - if the models change, they will fail, but the tested object may still be a s3bucket or an app. Are the facts that these methods assert relevant?